cypress/cypress/integration/record.spec.ts

```diff
@@ -2261,6 +2261,61 @@ describe("Record", () => {
     cy.get("[data-cy=trim-video]").should("not.exist");
   });
 
+  it("guide silhouette should be visible while trimming a video", () => {
+    cySetup(cy);
+    cyMockUpload(cy);
+    cyMockDefault(cy, {
+      mentor: [
+        videoMentor,
+        updateMentorAnswer(videoMentor, "A1_1_1", {
+          transcript: "My name is Clint Anderson",
+        }),
+      ],
+      gqlQueries: [
+        mockGQL("uploadTaskDelete", true, true),
+        mockGQL("updateAnswer", true, true),
+        mockGQL("updateQuestion", true, true),
+        mockGQL(
+          "uploadTasks",
+          [
+            [
+              {
+                question: {
+                  _id: videoMentor.answers[0].question._id,
+                  question: videoMentor.answers[0].question.question,
+                },
+                uploadStatus: "DONE",
+                transcript: "My name is Clint Anderson",
+                media: [
+                  {
+                    type: "video",
+                    tag: "web",
+                    url: "video.mp4",
+                  },
+                ],
+              },
+            ],
+          ],
+          true
+        ),
+      ],
+    });
+    cy.intercept("**/videos/mentors/*/*.mp4", { fixture: "video.mp4" });
+    cy.visit("/record");
+    cy.fixture("video.mp4").then((fileContent) => {
```

Comment:
...same comment about DRY violations. Fix globally for the project as follows:

```typescript
// in support functions...

export function cyAttachUpload(
  cy, 
  /* can also be parametrized to override default upload */
): Promise<void> {
  return cy.fixture("video.mp4").then((fileContent) => {
      cy.get('input[type="file"]').attachFile({
        fileContent: fileContent.toString(),
        fileName: "video.mp4",
        mimeType: "video/mp4",
      });
    });
}
```

...I made the above return a `Promise` because I suspect it's also a race condition to not wait for that async attach behavior to complete before continuing?

...so all usages should be more like

```typescript
cyAttachUpload(cy).then(() => {
  // all remaining test code here
})

Reply:
33259fb